IDEAS/Resources/domestichotwaterprofiles/README.md

```diff
@@ -0,0 +1,8 @@
+## Domestic Hot Water Profiles
+Two example profiles for DHW offtake are provided:
+- DHW_1year_2adults.txt (dwelling of 2 people: 2 adults)
```

Comment:
Maybe also add whether the adult types as specified in strobe?

Reply:
https://github.com/open-ideas/IDEAS/pull/1355/commits/a4f19b00926aeb49b1dddc8f23c794278f9f40cd